modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -13,7 +13,7 @@ configuration parameters.
 |===
 |Parameter |Description
 
-|`ManagementState`
+|`managementState`
 |`Managed`: The Operator updates the registry as configuration resources
 are updated.
```

Comment:
@IronicBadger To keep consistent, it's better to update other options too in this doc.
s/Logging/logging
s/HTTPSecret/httpSecret
s/Proxy/proxy
s/Storage/storage
s/ReadOnly/readOnly
s/Requests/requests
s/DefaultRoute/defaultRoute
s/Routes/routes
s/Replicas/replicas